website/docs/applications/manage_apps.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Manage applications
+---
+
+Managing the applications that your team uses involves several tasks, from initially adding the application and provider, to controlling access and visibility of the application, to providing access URLs.
+
+## Add new applications
+
+Learn how to add new applications from our video or follow the instructions below.
+
+### Video
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/broUAWrIWDI;start=22" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+### Instructions
+
+To add an application to authentik and have it display on users' **My applications** page, you can use the Application Wizard, which creates both the new application and the required provider at the same time.
+
+1. Log into authentik as an admin, and navigate to **Applications --> Applications**.
+
+2. Click **Create with Wizard**. (Alternatively, use our legacy process and click **Create**.)
```

Comment:
Suggestion: "As an alternative, start the legacy process by clicking **Create**.  The legacy process requires that the application and its authentication provider be configured separately."